packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -1,22 +1,66 @@
 'use strict';
 
 const get = require('lodash/get');
+const semver = require('semver');
+
 const { parseSQSMessageBody, sendSQSMessage } = require('@cumulus/aws-client/SQS');
 const log = require('@cumulus/common/log');
-const { getMessageExecutionArn } = require('@cumulus/message/Executions');
+const {
+  getKnexClient,
+  Executions,
+} = require('@cumulus/db');
+const {
+  getMessageExecutionArn,
+  getMessageExecutionParentArn,
+  getMessageCumulusVersion,
+} = require('@cumulus/message/Executions');
 const Execution = require('../models/executions');
 const Granule = require('../models/granules');
 const Pdr = require('../models/pdrs');
 const { getCumulusMessageFromExecutionEvent } = require('../lib/cwSfExecutionEventUtils');
 
-const saveExecutionToDb = async (cumulusMessage) => {
```

Comment:
This function seemed like a useless wrapper around the Dynamo write that just introduced the need for **more error handling and tests**, so I removed it